### PR TITLE
Yparikh/enum number

### PR DESCRIPTION
**Pull request by parikhyash**
@sushantk 

Handling numbers for enums.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: src/main/java/com/yahoo/xpathproto/JXPathCopier.java [hunk 1]
```diff
@@ -137,12 +137,15 @@ private JXPathCopier copyAsString(final Object sourceObject, final String target
     
     public JXPathCopier copyAsEnum(
         final Object sourceObject, final String targetField, final Descriptors.FieldDescriptor fieldDescriptor) {
-        if (sourceObject != null) {
```
Comment (sushantk) [inline src/main/java/com/yahoo/xpathproto/JXPathCopier.java]: we should still check for null sourceObject.

Comment (parikhyash) [inline src/main/java/com/yahoo/xpathproto/JXPathCopier.java]: done

**Comment by sushantk** [comment]
:+1:

**Pull request merged**